packages/xs/nbd.4.0.0b1/url

```diff
@@ -0,0 +1,2 @@
+archive: "https://github.com/xapi-project/nbd/archive/v4.0.0+beta1.tar.gz"
```

Comment:
The directory name should be exactly `package-name.package-version`, which is `4.0.0+beta1`, as described here, in the "repository format" section: http://opam.ocaml.org/doc/Manual.html#Layout